===================================================================
RCS file: /usr/OpenBSD/cvs/src/usr.bin/ssh/sshconnect.c,v
retrieving revision 1.147
retrieving revision 1.148
diff -u -r1.147 -r1.148
--- src/usr.bin/ssh/sshconnect.c	2003/06/29 12:44:38	1.147
+++ src/usr.bin/ssh/sshconnect.c	2003/09/18 07:52:54	1.148
@@ -267,9 +267,10 @@
 		optval = 0;
 		optlen = sizeof(optval);
 		if (getsockopt(sockfd, SOL_SOCKET, SO_ERROR, &optval, 
-		    &optlen) == -1)
+		    &optlen) == -1) {
 		    	debug("getsockopt: %s", strerror(errno));
 			break;
+		}
 		if (optval != 0) {
 			errno = optval;
 			break;
